src/GoogleFitOnFhir/Services/IImporterService.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace GoogleFitOnFhir.Services
 {
-    public interface IDataImporterService
+    public interface IImporterService
```

Comment:
Should this be named IImportRequestRouter? Just looking at the responsibility of this class, I think it behaves more like a router.

Reply:
Maybe.  Unlike the Identity/Authorize function, Publish isn't parsing a web request which to me is more akin to a router.  Publish is consuming a message from a queue.  If anything, the ResponsibilityHandler chain for Publish is the closest thing to a router.

Reply:
Naming. The most difficult problem in software... I'm fine with leaving as is.